From 6aa08d89f09837e237f264008877b4ffa10007d6 Mon Sep 17 00:00:00 2001
From: Robin Gong <B38343@freescale.com>
Date: Mon, 28 May 2012 11:19:27 +0800
Subject: [PATCH 1541/2463] ENGR00211142 mx6q : set VDDCORE 1.05V at 672Mhz
 cpu point

As SOC team suggestion, we should change VDDCORE from 1.1V to 1.05V at 672Mhz

Signed-off-by: Robin Gong <B38343@freescale.com>
---
 arch/arm/mach-mx6/cpu_op-mx6.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-mx6/cpu_op-mx6.c b/arch/arm/mach-mx6/cpu_op-mx6.c
index ac65b27..6fe2fd8 100644
--- a/arch/arm/mach-mx6/cpu_op-mx6.c
+++ b/arch/arm/mach-mx6/cpu_op-mx6.c
@@ -38,7 +38,7 @@ static struct cpu_op mx6_cpu_op_1_2G[] = {
 	{
 	 .pll_rate = 672000000,
 	 .cpu_rate = 672000000,
-	 .cpu_voltage = 1100000,},
+	 .cpu_voltage = 1050000,},
 	 {
 	  .pll_rate = 396000000,
 	  .cpu_rate = 396000000,
@@ -66,7 +66,7 @@ static struct cpu_op mx6_cpu_op_1G[] = {
 	{
 	 .pll_rate = 672000000,
 	 .cpu_rate = 672000000,
-	 .cpu_voltage = 1100000,},
+	 .cpu_voltage = 1050000,},
 	 {
 	  .pll_rate = 396000000,
 	  .cpu_rate = 396000000,
-- 
1.7.7.4

